test/address.test.js

```diff
@@ -219,6 +220,7 @@ define(function(require) {
             , body : undefined
             , context: document.body
             , origin: undefined
+            , timeout: 30000
```

Comment:
It looks like these are just added gratuitously and then never used, so they should be removed again.

Reply:
```timeout``` is passed to middleware using req, so we need to add them, else test case will fail.
 
Also, the test case has been updated for "configured web.req" to validate the timeout method with increased value.

Reply:
Ok, I will need to run these locally to check how this works, bear with me.